fractal-theme/assets/js/src/search-results.js

```diff
@@ -0,0 +1,75 @@
+import Fuse from 'fuse.js'
+
+function fractalSearchResults() {
+  const searchComponent = document.querySelector('.search');
+  const searchInput = document.getElementById('searchPageInput');
+  const searchResults = document.getElementById('searchResults');
+  const SearchApi = '/assets/theme/js/search.json';
+
+  // Search options
+  const searchOptions = {
+    keys: [{
+      name: 'data.title',
+      weight: 0.5
+    }, {
+      name: 'data.primaryKeywords',
+      weight: 0.4
+    }, {
+      name: 'data.secondaryKeywords',
+      weight: 0.1
+    }]
+  };
+
+  // Fetch endpoint
+  fetch(SearchApi)
+    .then(response => {
+      return response.json()
+    })
+    // Search init
+    .then(search => {
+      const fuse = new Fuse(search, searchOptions)
```

Comment:
Is it necessarily to create a new instance every time?

Reply:
Yes because it's two separate instances running at the same time :)